.env

```diff
@@ -32,7 +32,8 @@ NVMEOF_IPV6_ADDRESS=2001:db8::3
 NVMEOF_IO_PORT=4420
 NVMEOF_GW_PORT=5500
 NVMEOF_DISC_PORT=8009
-NVMEOF_EXPOSE_SERVICES="${NVMEOF_IO_PORT}/tcp:nvme,${NVMEOF_GW_PORT}/tcp:grpc,${NVMEOF_DISC_PORT}/tcp:nvme-disc"
+NVMEOF_PROMETHEUS_PORT=9202
```

Comment:
This port is already allocated to the Pacemaker exporter. I'd use a free port and register that in the [official Prometheus port allocation](https://github.com/prometheus/prometheus/wiki/Default-port-allocations).

Reply:
OK - switched to 10008, and port allocation registered.